manual/program/proposal_guidelines.rst

```diff
@@ -0,0 +1,56 @@
+Proposal Submission Guidelines
+==============================
+
+.. image:: tiptap.gif
+  :width: 400
+  :alt: Meme cat writing a SciPy proposal
+
+Here are the instructions and tips for submitting a proposal to SciPy.
+
+Instructions
+------------
+
+.. image:: tiptappy.gif
+  :width: 400
+  :alt: Another meme cat writing a SciPy proposal
+
+**Abstract**
+
+Your Abstract will appear in the online schedule and give attendees a sense of your talk. This should be around 100 words or less.
+
+**Description**
+
+Your placement in the program will be based on reviews of your description. This should be a roughly 500-word outline of your presentation. This outline should concisely describe software of interest to the SciPy community, tools or techniques for more effective computing, or how scientific Python was applied to solve a research problem. A traditional background/motivation, methods, results, and conclusion structure is encouraged but not required. Links to project websites, source code repositories, figures, full papers, and evidence of public speaking ability are encouraged.
+
+**Format**
```

Comment:
```suggestion
**Notes** 
The notes section of the CFP will not be made public like the abstract and description are. Use the notes section to include miscellaneous information that you think may be helpful to reviewers, or to make a note about how you expect your project to change between the time of submission and SciPy. 

**Format**
```

Reply:
People had questions about the notes part of the CFP in office hours so i've added a description here.